packages/web/src/common/store/cache/users/sagas.js

```diff
@@ -407,7 +407,8 @@ export function* fetchUserSocials({ handle }) {
           tiktok_handle: socials.tikTokHandle || null,
           website: socials.website || null,
           donation: socials.donation || null,
-          _artist_pick: socials.pinnedTrackId || null
+          _artist_pick: socials.pinnedTrackId || null,
+          artist_pick_track_id: socials.pinnedTrackId | null
```

Comment:
Set `artist_pick_track_id` in the user metadata with the value from the identity service when populating the user in the cache. This is necessary for the case during the migration where, after backfilling the users table in discovery, there is a track id in the users table but `artist_pick_track_id` in the cache is null. Upon saving an update to user metadata, the backfilled artist pick track id in the users table would be incorrectly overwritten.